### PR TITLE
Guya stubs

### PR DESCRIPTION
**Pull request by eu-genia**
stubs from the spreadsheet by Guesh

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
In my opinion GT is too short and may easily coincide with or come too close to another shelfmark. Perhaps GUTH would work out better.

**Comment by eu-genia** [comment]
at the moment GT is not assigned as far as I can see, and this is the shelfmark visible on the photos

**Review by GueshSol — APPROVED** [review]

File: GuyaTH/GT001.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                            <decoNote xml:id="d2"></decoNote>
+                            <decoNote xml:id="d3"></decoNote>
+                            <decoNote xml:id="d4"></decoNote>
+                            <decoNote xml:id="d5"></decoNote>
+                            
+                        </decoDesc>
+                        <additions><list>
+                                <item xml:id="a1"><desc>the manuscript contains short Ge'ez notes provided as colophons and landmark notes.
+                                    </desc></item>
+                            <item xml:id="a2"><desc>the manuscript contains short Greek,
```
Comment (thea-m) [inline GuyaTH/GT001.xml:73]: Really Greek? If so, to be added among the languages

File: GuyaTH/GT005.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="17"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic"/>
+                        </handDesc>
+                        <decoDesc><decoNote xml:id="d1" type="drawing"><desc>The manuscript contains several talismanic images insrted in between the texts
```
Comment (thea-m) [inline GuyaTH/GT005.xml:60]: inserted

File: GuyaTH/GT007.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                                <layout columns="3" writtenLines="33"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic"/>
+                        </handDesc>
+                        <decoDesc><decoNote xml:id="d1" ><desc>harags</desc></decoNote></decoDesc>
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">Wooden boards coverd by laether and protecting textile
```
Comment (thea-m) [inline GuyaTH/GT007.xml:64]: leather

**Review by thea-m — CHANGES_REQUESTED** [review]
Very nice!
I personally feel that it would make a much better impression of both Bm and the PhD project if already for such stubs the transcriptions were correct :)

File: GuyaTH/GT001.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply GuyaTH/GT001.xml:73]: Not Greek, there is another issue, of "cryptic writing" (appeared in the field report)

File: GuyaTH/GT001.xml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                                <decoNote xml:id="b1">Two wooden boards, covered with
+                                    cloth</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1700" notAfter="1800">18th-century (colophon) </origDate></origin>
+                        <provenance>The manuscript was written for the eighteenth-century Gondarine
+                            king, Bakaffa. At some epoch, the manuscript became part of the Guyā
+                            Takla Haymanot collection. </provenance>
```
Comment (thea-m) [inline GuyaTH/GT001.xml:89]: Hāymānot

File: GuyaTH/GT004.xml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/004/" n="134">GT-004</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Gadla Takla Haymānot</title>
```
Comment (thea-m) [inline GuyaTH/GT004.xml:36]: Hāymānot

File: GuyaTH/GT005.xml [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/005/" n="80">GT-005</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Maftǝhe Sǝray</title>
```
Comment (thea-m) [inline GuyaTH/GT005.xml:36]: Maftǝḥe Śǝrāy

File: GuyaTH/GT006.xml [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/006/" n="64">GT-006</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Ra'ya Maryam
```
Comment (thea-m) [inline GuyaTH/GT006.xml:36]: Rāʾya Māryām

File: GuyaTH/GT007.xml [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/007/" n="173">GT-007</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Dǝggwā
```
Comment (thea-m) [inline GuyaTH/GT007.xml:36]: Dǝggʷā

File: GuyaTH/GT008.xml [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/008/" n="70">GT-008</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
```
Comment (thea-m) [inline GuyaTH/GT008.xml:35]: Gadla ʿAbiya ʾƎgziʾ

File: GuyaTH/GT009.xml [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/009/" n="64">GT-009</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Rǝtua Ḥāymānot</title>
```
Comment (thea-m) [inline GuyaTH/GT009.xml:36]: Rǝtuʿa Ḥāymānot

File: GuyaTH/GT010.xml [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/010/" n="106">GT-010</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Aʿǝmada mǝsṭir
```
Comment (thea-m) [inline GuyaTH/GT010.xml:36]: ʾAʿmāda mǝśṭir

File: GuyaTH/GT011.xml [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0011GuyaTH"/>
+                        <idno facs="GT/011/" n="155">GT-011</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title>Gadla Takla Haymānot</title>
```
Comment (thea-m) [inline GuyaTH/GT011.xml:36]: Hāymānot

**Review by thea-m — CHANGES_REQUESTED** [review]
I am not sure whether some of these transcribe the actual variants of the manuscripts, so did not correct in the files.

File: GuyaTH/GT001.xml [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+                            <decoNote xml:id="d1"><desc>decorated in the so-called Second Gondarine style
+                            </desc></decoNote>
+                            <decoNote xml:id="d2"></decoNote>
+                            <decoNote xml:id="d3"></decoNote>
+                            <decoNote xml:id="d4"></decoNote>
+                            <decoNote xml:id="d5"></decoNote>
+                            
+                        </decoDesc>
+                        <additions><list>
+                                <item xml:id="a1"><desc>short Ge'ez notes provided as colophons and landmark notes.
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply GuyaTH/GT001.xml:71]: I suggest that further corrections and adjustments are carried out by @GueshSol - dear @GueshSol please switch to this branch and edit the records where necessary, thank you!

**Comment by eu-genia** [comment]
@GueshSol can these records be fixed? this pull request has not been merged so they are not visible from outside, they must be edited on the branch first.

**Comment by eu-genia** [comment]
In your GitHub you should switch to Guya branch and edit the records.
Ambǝrsǝwā and Juwāmara pull requests have been merged but Guya has not because there are edits expected.
image

**Comment by GueshSol** [comment]
Okay. Thank you so much!

**Comment by eu-genia** [comment]
Thank you, Guesh, please do not also forget that you have not yet submitted the images. Thank you.

**Pull request merged**